src/state/slices/opportunitiesSlice/resolvers/foxfarming.ts

```diff
@@ -80,13 +92,16 @@ export const foxFarmingLpMetadataResolver = async ({
   // Amount of Eth in liquidity pool
   const ethInReserve = bnOrZero(reserves?.[0]?.toString()).div(`1e${ethPrecision}`)
 
+  // TODO(gomes): This is a hackish way we were previously doing this, getting the ETH liquidity * 2 will get us close to the real fiat TVL but we can do better
```

Comment:
Fine for now but definitely something to clean up since we can derive fiat values for token0 and token1